pkg/asset/installconfig/gcp/validation.go

```diff
@@ -16,6 +16,7 @@ import (
 	"k8s.io/apimachinery/pkg/util/validation/field"
 
 	"github.com/openshift/installer/pkg/types"
+	typesdns "github.com/openshift/installer/pkg/types/dns"
```

Comment:
Nit: to keep it consistent with how you imported it in other parts
```suggestion
	dnstypes "github.com/openshift/installer/pkg/types/dns"
```